### ISSUE TITLE
gptqlora seems to make the model worse than gptq.

### ISSUE THREAD

**Issue by jianyuheng** [taprosoft/llm_finetuning#3]
I’m trying to finetune my gptq  pretrained model via your framework, but I found that gpt+lora seems to make the model worse.

During the initialization phase of training, I met the warning: 
The safetensors archive passed at gptq_model-4bit-128g.safetensors does not contain metadata. Make sure to save your model with the `save_pretrained` method. Defaulting to 'pt' metadata.

hope you can give me some advice, thanks!

**Comment by taprosoft** [comment]
Can you explain more about your issue. You can try to run the evaluation loop first to see the loss of the model before and after training.
The warning above is normal and expected. It will not affect the training process.

**Comment by jianyuheng** [comment]
Sorry! I found some bugs in my code, I will close this issue.

**Comment by jianyuheng** [comment]
checked.
